packages/native/src/TxNative.js

```diff
@@ -124,9 +124,11 @@ export default class TxNative {
         translation = this.missingPolicy.handle(translation, this.currentLocale);
       }
 
+      if (!isString(translation)) translation = `${translation}`;
       return translation;
     } catch (err) {
-      return this.errorPolicy.handle(err, sourceString, this.currentLocale, params);
+      return this.errorPolicy.handle(err,
+        `${sourceString}`, this.currentLocale, params);
```

Comment:
Hm, eslint didn't have  problem with this?

Reply:
You mean about the indentation or something else? In general, it didn't.